README.md

```diff
@@ -22,37 +22,37 @@ It's used in the [snap framework](http://github.com:intelsdi-x/snap).
 * [golang 1.5+](https://golang.org/dl/)
 
 ### Operating systems
-All OSs currently supported by snap:
+All OSs currently supported by Snap:
 * Linux/amd64
 
 ### Installation
-#### Download load plugin binary:
-You can get the pre-built binaries for your OS and architecture at snap's [GitHub Releases](https://githubYou can get the pre-built binaries for your OS and architecture at snap's [GitHub Releases](https://github.com/intelsdi-x/snap/releases) page. Download the plugins package from the latest release, unzip and store in a path you want `snapd` to access.
+#### Download the plugin binary:
+
+You can get the pre-built binaries for your OS and architecture from the plugin's [GitHub Releases](https://github.com/intelsdi-x/snap-plugin-collector-load/releases) page. Download the plugin from the latest release and load it into `snapd` (`/opt/snap/plugins` is the default location for Snap packages).
 
 #### To build the plugin binary:
-Fork https://github.com/intelsdi-x/snap-plugin-collector-load  
+
+Fork https://github.com/intelsdi-x/snap-plugin-collector-load
```

Comment:
This line is joined with the next line, please add a new line after this sentence.